client/add/add.ctrl.js

```diff
@@ -123,10 +123,27 @@ module.exports = class Add {
            toastAction,
            toastAction ? `/pokemon/${successfulUploads[0].created.id}` : undefined);
       })
-      .filter(line => line.success && line.created.box === box.data.id)
-      .then(lines => lines.slice(0, BOX_PAGE_SIZE - (box.data.contents.length % BOX_PAGE_SIZE)))
+      .filter(line => line.success)
       .map(line => line.created)
-      .then(lines => box.data.contents.push(...lines))
+      .reduce((overall, line) => {
```

Comment:
Yeah, good point. When writing the code, I couldn't figure out what to name it so I called it overall as a temp name and forgot to change it.